src/hooks/useTokenIdentityRegistry.ts

```diff
@@ -6,7 +6,7 @@ import {
   watchIdentityRegistryIdentityRegisteredEvent,
 } from "@/services/contracts/wagmiGenActions";
 
-export function useTokenIdentityRegistry(tokenSelected: TokenNameItem | null) {
+export function useTokenIdentityRegistry(tokenSelected?: TokenNameItem) {
```

Comment:
Why `tokenSelected` became optional? Whats the case for the call without a token?

Reply:
ah, I see it stems from this definition, makes sense

```
  const [tokenSelected, setTokenSelected] = useState(
    null as TokenNameItem | null,
  );
```